src/_includes/macros/share.html

```diff
@@ -107,14 +107,14 @@
     } %}
   {% set _ignore = settings.update(options) %}
 
-    <aside class="share
+    <div class="share
                  {{ settings.additional_classes
                  if settings.additional_classes }}">
-        <h1 class="h6 share_heading
+        <span class="h6 share_heading
```

Comment:
It breaks things. Sometimes share is under an h1, h2 or even an h3. Setting it in the outline without the h5o risks breaking the outline for something that really isn't that important in the site hierarchy.